src/app/modules/auth/dtos/RegisterDTO.js

```diff
@@ -0,0 +1,26 @@
+class RegisterDTO {
```

Comment:
Represents a request to the register route. Basically just encapsulates the data we care about from the request and prevents us from passing the entire request object all over the codebase. Also contains the schema which is used in the server to validate the incoming request (will send a 400 response if invalid)